torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3138,6 +3157,32 @@ def sample_inputs_entr(op_info, device, dtype, requires_grad, **kwargs):
                                     low=low,
                                     requires_grad=requires_grad)))
 
+# TODO: Consolidate `i0e` with sample_inputs_unary when `make_tensor`,
+#       supports `exclude` argument.
+#       For more context: https://github.com/pytorch/pytorch/pull/56352#discussion_r633277617
+def sample_inputs_i0_i1(op_info, device, dtype, requires_grad, **kwargs):
+
+    samples = (SampleInput(make_tensor((S,), device, dtype,
```

Comment:
If the op is torch.special.i0e, doesn't this and the next sample input need to exclude zero?

Reply:
Ideally yes. But at this point we don't have `exclude_zero` yet. Temporarily I will add `x[x == 0] = eps` and add a comment to replace it with `exclude_zero` once that is in. Is that ok?

Reply:
Great idea